src/apps/components/AppPermissions/AppPermissions.tsx

```diff
@@ -0,0 +1,49 @@
+import { AppsListQuery } from "@saleor/graphql";
+import {
+  IconButton,
+  makeStyles,
+  PermissionsIcon,
+  Tooltip
+} from "@saleor/macaw-ui";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const useStyles = makeStyles(
+  () => ({
+    list: {
+      margin: 0,
+      paddingLeft: "16px"
+    }
+  }),
+  { name: "AppPermissions" }
+);
+
+interface AppPermissionsProps {
+  permissions: AppsListQuery["apps"]["edges"][0]["node"]["permissions"];
```

Comment:
Permission should already be a fragment (if it's not, add one) so you can use the generated type from that